test/ArborX.spec

```
@@ -0,0 +1,126 @@
+# Stolen from eigen spec
+# The (empty) main package is arch, to have the package built and tests run
+# on all arches, but the actual result package is the noarch -devel subpackge.
+# Debuginfo packages are disabled to prevent rpmbuild from generating an empty
+# debuginfo package for the empty main package.
+%global debug_package %{nil}
+
+Name:           ArborX
+Version:        2.0
+%global         sover 0
+Release:        1%{?dist}
+Summary:        Performance-portable geometric search library
+# no support for 32-bit archs https://github.com/kokkos/kokkos/issues/2312
+# mpi is broken on s390x see: bug#2322073 
+ExcludeArch: i686 armv7hl s390x
+
+License:        BSD-3-Clause
+URL:            https://github.com/arborx/%{name}
+Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  gcc-c++
+BuildRequires:  cmake >= 3.16
+BuildRequires:  kokkos-devel
+BuildRequires:  openmpi-devel
+BuildRequires:  mpich-devel
+BuildRequires:  boost-devel
+BuildRequires:  google-benchmark-devel
+
+%global arborx_desc \
+ArborX is an open-source library designed to provide performance portable \
+algorithms for geometric search, similarly to nanoflann and Boost Geometry.
+
+%description
+%{arborx_desc}
+
+%package devel
+Summary:        Development package for  %{name} packages
+BuildArch:      noarch
+%description devel
+%{arborx_desc}
+
+This package contains the development files of %{name}.
+
+%package openmpi-devel
+Summary:    openmpi development headers and libraries for %{name}
+BuildArch:  noarch
+Requires:   openmpi-devel
+
+%description openmpi-devel
+%{arborx_desc}
+
+This package contains openmpi development files of %{name}.
+
+%package mpich-devel
+Summary:    mpich development headers and libraries for %{name}
+BuildArch:  noarch
+Requires:   mpich-devel
+
+%description mpich-devel
+%{arborx_desc}
+
+This package contains mpich development files of %{name}.
+
+%prep
+%autosetup -p1
+
+%build
+# save memory
+%global _smp_mflags -j1
+%global _vpath_builddir %{_target_platform}-${mpi:-serial}
+
+for mpi in '' mpich openmpi; do
```

As the issue happens in serial, we could do for now:
```suggestion
for mpi in ''; do
```

---

I had to undo this, not 100% sure.